ContentTransformer/HtmlStylesRemover.php

```diff
@@ -62,18 +65,33 @@ private function getInnerNode(\DOMNode $node): \DOMNode
 
     protected function initDocument(string $input): \DOMDocument
     {
-        $doc = new \DOMDocument();
+        $doc = new \DOMDocument('1.0', 'UTF-8');
         $doc->preserveWhiteSpace = false;
         $doc->formatOutput = true;
 
-        $doc->loadHtml(
-            $this->addTemporaryWrapper($input),
+        @$doc->loadHtml( // Need @ to bypass the htmlParseEntityRef warnings/errors
+            \mb_convert_encoding($this->addTemporaryWrapper($input), 'HTML-ENTITIES', 'UTF-8'),
```

Comment:
Which kind of error do you face? Is not not better to mb_emcode the  $input? Or to to do this convert in the addTemporaryWrapper?

Reply:
@theus77  It's for avoid this kind of warning :  "Warning: DOMDocument::loadHTML(): htmlParseEntityRef: expecting ';' in Entity, line: 15". 

It's seems to be a knowing workaround used by Drupal and even Symfony (https://github.com/symfony/dom-crawler/blob/master/Crawler.php#L1128)